docs/content/installation/configuration.md

```diff
@@ -276,3 +294,92 @@ for pgBackRest and pgBouncer respectively
 | `POD_ANTI_AFFINITY` | preferred |  | This will set the default pod anti-affinity for the deployed PostgreSQL clusters. |
 | `POD_ANTI_AFFINITY_PGBACKREST` |  |  | This will set the default pod anti-affinity for the pgBackRest pods. |
 | `POD_ANTI_AFFINITY_PGBOUNCER` |  |  | This will set the default pod anti-affinity for the pgBouncer pods. |
+
+## Resource Configuration
```

Comment:
I'm not sure this section is still valid. I don't see references to these resource configuration variables anywhere else in the code. Does anyone know where its used or can I remove it?

Reply:
@jmckulk Yup, please remove.